src/ripple/app/misc/impl/TxQ.cpp

```diff
@@ -1599,7 +1599,7 @@ setup_TxQ(Config const& config)
 std::unique_ptr<TxQ>
 make_TxQ(TxQ::Setup const& setup, beast::Journal j)
 {
-    return std::make_unique<TxQ>(setup, std::move(j));
+    return std::make_unique<TxQ>(setup, j);
```

Comment:
Since you're here: this `make_TxQ` function is entirely unnecessary; we usually use this idiot when we have an interface class and the implementation is hidden and inaccessible. This isn't the case here, so it could go away by directly inlining the `std::make_unique<TxQ>(setup, j)` in **Application.cpp**

Reply:
Fixed in `5087498fd [fold] Rm make_TxQ factory function`

Lol'd at the mistype of `idiot` for `idiom`

Reply:
LOL... I just noticed that typo. :facepalm: